jfix-stdlib-ratelimiter/src/main/kotlin/ru/fix/stdlib/ratelimiter/SwitchableRateLimitedDispatcher.kt

```diff
@@ -0,0 +1,88 @@
+package ru.fix.stdlib.ratelimiter
+
+import ru.fix.aggregating.profiler.Profiler
+import ru.fix.dynamic.property.api.DynamicProperty
+import java.util.concurrent.CompletableFuture
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.atomic.AtomicReference
+import java.util.function.Supplier
+import kotlin.coroutines.CoroutineContext
+
+class SwitchableRateLimitedDispatcher(
```

Comment:
I imagined it in other simplier way.

> it will depend on ready-to-use instances of both types of RateLimitedDispatcher

> will switch between implementations depending on dynamic bool flag value

We can just depend on ready-to-use instances of RateLimitedDispatcherInterface and switch between them based on some dynamic prop boolean flag. 
In which functions we will check this flag and which ones we will access both dispatchers?